tgstation.dme

```diff
@@ -1,6 +1,8 @@
 // DM Environment file for tgstation.dme.
 // All manual changes should be made outside the BEGIN_ and END_ blocks.
 // New source code should be placed in .dm files: choose File/New --> Code File.
+#include "code\genesis_call.dme"
```

Comment:
The .dme will not stay ordered like this by SpacemanDMM, DreamMaker, etc. You may have to rename to file to `/__genesis_call.dm`.

Reply:
That's why it's genesis_call.**dme**, not genesis_call.**dm**

Reply:
already tested, both dreammaker and spacemandmm don't touch it, likely bc it's a dme and not a dm